data/src/main/java/com/architectcoders/data/repository/MarvelRepository.kt

```diff
@@ -11,32 +11,44 @@ class MarvelRepository(
     private val remoteDataSource: RemoteDataSource,
     private val localDataSource: LocalDataSource,
 ) {
-    suspend fun getLocalCharacters(): List<CharacterResult>? {
-        return localDataSource.getLocalCharacters()
-    }
+    suspend fun getLocalCharacters(): List<CharacterResult>? =
+        localDataSource.getLocalCharacters()
 
-    suspend fun getRemoteCharacters(offset: Int): Characters {
-        return remoteDataSource.getRemoteCharacters(offset)
-    }
+    suspend fun getRemoteCharacters(offset: Int): Characters =
+        remoteDataSource.getRemoteCharacters(offset)
 
-    suspend fun getRemoteCharacterById(characterId: Int): Characters =
-        remoteDataSource.getRemoteCharacterById(characterId)
+    suspend fun getStoredCharactersCount(): Int =
+        localDataSource.getStoredCharactersCount()
+
+    suspend fun getRemoteCharacterById(characterId: Int): CharacterResult =
```

Comment:
Change name of function by getLocalCharacterById

Reply:
Sure :)